README.md

```diff
@@ -65,6 +65,21 @@ Tests for this contract are implemented in Haskell using the [cleveland framewor
 For more detail about the test suite and how to execute it, check out its
 [README](./haskell/test/README.md).
 
+## Contract Metadata
+
+BaseDAO metadata should be compliant with [TZIP-16](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md).
+
+It can be generated via:
+
+```bash
+cd haskell
+stack exec -- baseDAO-ligo-meta print-metadata > metadata.json
+```
+
+This metadata should be stored via:
+- As a separate contract: [The `tezos-storage` URI Scheme](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-16/tzip-16.md#the-tezos-storage-uri-scheme)
+- On IPFS file storage: [Get started with IPFS](https://ipfs.io/)
+
```

Comment:
I think one thing is missing here: the step, which is how to use this stored metadata

I think we should point at the right section of the TZIP-16 spec and perhaps even quote the main part:
> the encoding of the values must be the direct stream of bytes of the data being stored.
> For instance, an URI starting with `http:` will start with the 5 bytes `0x687474703a` (`h` is `0x68`, `t` is `0x74`, etc.). 

and after this conversion mention that the `metadata_map` can be used (point at the `building` doc).

Reply:
I've added this now, as part of the `building.md`.